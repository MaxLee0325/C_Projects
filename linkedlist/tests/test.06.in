new
append alpha
append beta
append gamma
iterator
next
next
set delta
end
