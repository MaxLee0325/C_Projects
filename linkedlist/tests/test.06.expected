[ ]
[ alpha ]
[ alpha beta ]
[ alpha beta gamma ]
[ alpha beta gamma ]
Next: alpha
[ alpha beta gamma ]
Next: beta
[ alpha beta gamma ]
Set: 'beta' -> 'delta'
[ alpha delta gamma ]
